config/cloud-config/master-config-template.yaml

```diff
@@ -20,17 +20,18 @@ write_files:
         echo "Kubenetes not installed - installing."
 
         # Get the Kubernetes Release.
-        wget https://github.com/kubernetes/kubernetes/releases/download/v1.1.0/kubernetes.tar.gz
-        
+        wget https://github.com/kubernetes/kubernetes/releases/download/v1.1.1/kubernetes.tar.gz
+
         # Extract the Kubernetes binaries.
         tar -xf kubernetes.tar.gz
         tar -xf kubernetes/server/kubernetes-server-linux-amd64.tar.gz
-        
+
         # Install the `kubelet` and `kubectl` binaries.
         sudo chmod +x kubernetes/server/bin/kubelet
         sudo chmod +x kubernetes/server/bin/kubectl
         sudo cp -f kubernetes/server/bin/kubelet /opt/bin
         sudo cp -f kubernetes/server/bin/kubectl /opt/bin
+        sudo rm -rf /kubernetes
```

Comment:
no, but its a huge directory and we need 2 files from it. I did the same for our calico-kubernetes clone too.